docs/roadmap.md

```diff
@@ -52,48 +52,9 @@ We also ship experimental features as we develop them. You can read more about t
-| Fluent Shadows                                                        | &#128994;          | &#128994;          | &#128994;      | &#128994;     | &#128994; | &#128994;
-| Input validation for data fields                                      | &#128311;          | &#128311;          | &#128311;      | &#128311;     | &#128311; | &#128311;
-| Supports multiple top-level windows on the same thread                | &#128311;          | &#128311;          | &#128994;[^2]  | &#128994;     | &#128994; | &#128994;
-| Supports multiple top-level windows on separate threads               | &#128311;          | &#128311;          | &#128311;      | &#128311;     | &#128311; | &#128311;
-| Drag and drop                                                         | &#128994;          | &#128994;          | &#128994;      | &#128994;     | &#128994; | &#128994;
-| RenderTargetBitmap                                                    | &#128994;          | &#128994;          | &#128994;      | &#128994;     | &#128994; | &#128994;
-| Mouse cursor customization                                            | &#128994;          | &#128994;          | &#128994;      | &#128994;     | &#128994; | &#128994;
-| Animated Gif support                                                  | &#128994;          | &#128994;          | &#128994;      | &#128994;     | &#128994; | &#128994;
-| VirtualSurfaceImageSource (VSIS) support                              | &#128994;          | &#128994;          | &#128994;      | &#128994;     | &#128994; | &#128994;
-| In-app acrylic                                                        | &#128994;          | &#128994;          | &#128994;      | &#128994;     | &#128994; | &#128994;
-| XAML Islands                                                          |                    | &#128311;          | &#128311;      | &#128311;     | &#128311; | &#128311;
-| Mica                                                                  |                    |                    |                | &#128994;     | &#128994; | &#128994;
-| Widgets                                                               |                    |                    |                |               | &#128994; | &#128994;
-| Media playback controls                                               |                    |                    |                |               | &#128994;  | &#128994;
-| InfoBadge and other updates from WinUI 2.8                            |                    |                    |                |               | &#128994; | &#128994;
-
-[^1]: Supported on Visual Studio 2022 Preview 5 and above for packaged apps only 
-[^2]: Supported in 1.0.1 and above
+### Roadmap
+
+For more info about upcoming features in the next release of WinUI 3, see the [Windows App SDK feature roadmap](https://github.com/microsoft/WindowsAppSDK/blob/main/docs/roadmap.md).
```

Comment:
The page is outdated.

Reply:
Correct, we are planning on updating it very soon with fresh information about what's coming next for WinUI 3. Stay tuned! :)